app/views/shared/dialogs/_dialog_field_drop_down_list.html.haml

```diff
@@ -4,9 +4,8 @@
   = select_tag(field.name, options_for_select(select_values, selected), drop_down_options(field, url))
 
   :javascript
-    miqInitSelectPicker();
-    $('.selectpicker').selectpicker('val', '#{selected}');
-    miqSelectPickerEvent('#{field.name}', '#{url}');
+    var selectedValue = $('select[name="#{field.name}"]').val();
+    dialogFieldRefresh.initializeDialogSelectPicker('#{field.name}', '#{field.id}', selectedValue, '#{url}');
```

Comment:
I think we want to use `#{selected}` here instead of re-deriving the selectedValue, because `selected` should be coming from the server for when something gets refreshed.